sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/specialized/BlobClientBase.java

```diff
@@ -1013,6 +1013,59 @@ public BlobDownloadContentResponse downloadContentWithResponse(
         return blockWithOptionalTimeout(download, timeout);
     }
```

Comment:
The existing downloadContentWithResponse can call into the new downloadContentWithResponse by passing getRangeContentMd5 as false.